src/librustc_mir/const_eval.rs

```diff
@@ -632,7 +632,11 @@ pub fn const_eval_raw_provider<'tcx>(
     }
 
     let span = tcx.def_span(cid.instance.def_id());
-    let mut ecx = InterpretCx::new(tcx.at(span), key.param_env, CompileTimeInterpreter::new());
+    let mut ecx = InterpretCx::new(
+        tcx.at(span),
+        key.param_env,
+        CompileTimeInterpreter::new(),
+        Default::default());
```

Comment:
Please follow rustfmt: The final `)` should be on its own line, and less indented.

Reply:
Oh ok sorry, I don't know why my text editor is messing the identation.

Reply:
It didn't, the parenthesis wasn't even on its own line. ;)